react-list.es6

```diff
@@ -113,7 +116,14 @@ module.exports = class ReactList extends Component {
       }, 0);
     }
 
-    this.updateFrame();
+    if (type === 'simple' && stableFrameDelay) {
```

Comment:
I don't think we need to restrict this prop to the `'simple'` list type. It seems like it could potentially be used for `'variable'` or `'uniform'` as well.